src/nmodl/noccout.cpp

```diff
@@ -241,7 +243,8 @@ void c_out() {
         P("\nstatic void nrn_jacob(_nrn_model_sorted_token const& _sorted_token, NrnThread* "
           "_nt, Memb_list* _ml_arg, int _type) {\n");
         P("_nrn_mechanism_cache_range _lmr{_sorted_token, *_nt, *_ml_arg, _type};\n");
-        P("auto* const _vec_d = _nt->node_d_storage();\n");
+        P("auto const _vec_d = _nt->node_d_storage();\n");
```

Comment:
`auto*` -> `auto` is an incidental change that is not immediately needed in this PR.
The logic is that it allows `node_d_storage()` etc to seamlessly change from returning `T*` to returning `std::span<T>` in future.